website/versioned_docs/version-0.1.0/architecture/frontends/snowflake.md

```
@@ -0,0 +1,191 @@
+
+cur = conn.cursor()
+cur.execute("SELECT 1 AS num, 'hello' AS greeting")
+for row in cur:
+    print(row)
+```
+
+The Python connector sends requests to the wire protocol endpoints. Set `host` and `port` to point at the QueryFlux Snowflake listener. The `account` value is required by the driver but not used by QueryFlux for routing.
+
+### JDBC
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)